distributed/preloading.py

```diff
@@ -131,9 +132,23 @@ def _download_module(url: str) -> ModuleType:
     logger.info("Downloading preload at %s", url)
     assert is_webaddress(url)
 
-    request = urllib.request.Request(url, method="GET")
-    response = urllib.request.urlopen(request)
-    source = response.read().decode()
+    duration = 0.2
+    for i in range(10):
```

Comment:
This would probably work but i'd really like to adjust how persistent it would be. For our application we'd probably want about 2-3 minutes of retrying if needed

Reply:
Perhaps this could be made configurable?

Reply:
I'm happy to do whatever.  I'm curious though, under what conditions would a web application not respond under 3 minutes?  That seems like a long time to me.  If that application is Coiled, then I'd love to see higher standards applied there if possible rather than making Dask more lenient than is typical.

Reply:
I don't have hard data, but I suspect there's a long tail of events not-entirely-under-our-control that would justify retrying.

E.g. AWS networking has a hiccup for a little while, etc.

I'm not sure if that really justifies long retries, though.